pipeline/terraform/main.tf

```diff
@@ -0,0 +1,194 @@
+provider "aws" {
+  profile = "${var.profile}"
+  region  = "${var.region}"
+}
+
+locals {
+  network_prefix = "10.1"
+  availability_zones = [ "${var.region}a", "${var.region}b" ]
+  resource_label = "${var.project}-${terraform.workspace}"
+}
+
+// ~~ Nginx container ~~~
+
+data "template_file" "nginx_task_definition" {
```

Comment:
Is this intended to be replaced with Java/spring container?  Or stuff java app within container?  Not picking up the reason for using nginx container.

Reply:
yes, nginx is just a placeholder that was used to verify that the networking and security groups were set up correctly. The application load balancer should be capable of whatever proxy capabilities will be needed and this initial container will be replaced with Jenkins, Tomcat etc. in future commits.

Reply:
Alright, then we will need an ECR too to push images.  I am sure that is in your thought process too.